src/MongoDB.Driver/FilterDefinitionBuilder.cs

```diff
@@ -796,6 +796,26 @@ public FilterDefinition<TDocument> In<TField>(Expression<Func<TDocument, TField>
             return In(new ExpressionFieldDefinition<TDocument, TField>(field), values);
         }
 
+        /// <summary>
+        /// Creates a json schema validator.
+        /// </summary>
+        /// <param name="schema">The json validation schema.</param>
+        /// <returns>An schema validator.</returns>
+        public FilterDefinition<TDocument> JsonSchema(string schema)
+        {
+            return JsonSchema(BsonDocument.Parse(schema));
+        }
+
+        /// <summary>
+        /// Creates a json schema validator.
+        /// </summary>
+        /// <param name="schema">The json validation schema.</param>
+        /// <returns>An schema validator.</returns>
+        public FilterDefinition<TDocument> JsonSchema(BsonDocument schema)
+        {
+            return new BsonDocument("$jsonSchema", schema);
```

Comment:
For clarity consider:

    return new BsonDocumentFilterDefinition<TDocument>(new BsonDocument("$jsonSchema", schema));